bundle/ezpublish_legacy/ngadminui/design/ngadminui/templates/content/datatype/edit/ezgmaplocation.tpl

```diff
@@ -9,7 +9,7 @@
 
     <div class="col-sm-8">
         {run-once}
-        <script type="text/javascript" src="//maps.google.com/maps/api/js?sensor={ezini('GMapSettings', 'UseSensor', 'ezgmaplocation.ini')}"></script>
+        <script type="text/javascript" src="//maps.google.com/maps/api/js?key={ezini('GMapSettings', 'ApiKey', 'ngadminui.ini')}"></script>
```

Comment:
What's wrong with `ezgmaplocation.ini` ? This datatype comes from `ezgmaplocation` extension, after all.